metaserver/src/main/scala/scala/meta/languageserver/compiler/CompletionProvider.scala

```diff
@@ -32,11 +33,28 @@ object CompletionProvider extends LazyLogging {
         isUsedLabel += label
         items += CompletionItem(
           label = label,
-          detail = Some(r.sym.signatureString)
+          detail = Some(r.sym.signatureString),
+          kind = completionItemKind(compiler)(r)
         )
       }
     }
     CompletionList(isIncomplete = false, items = items.result())
   }
 
+  private def completionItemKind(
+      c: Global
+  )(r: c.CompletionResult#M): Option[Int] = {
```

Comment:
Would be nice if we could to get rid of that `Option[Int]` in favor of a wrapper class 

```scala
case class CompletionItemKind(id: Int)
object CompletionItemKind {
  implicit val writes: Writes[CompletionItemKind] = new Writes[CompletionItemKind] {
    override def writes(o: CompletionItemKind) = JsNumber(o.id)
  }
```

Not blocking in this PR, but maybe in the future.